blog-post.html

```diff
@@ -4,9 +4,41 @@
     <title>Blog Post</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    <link href="/style2.css" rel="stylesheet" type="text/css">
+  
   </head>
+  <style>
+    a {color: Black;}
+    a:hover {color: yellow};
+  </style>

```

Comment:
This is fine to put here, but since you do have a linked CSS sheet, you could put this into there to keep your CSS all in the same space